### PR TITLE
Space Traveler's Hub Full Project

### PR DESCRIPTION
**Pull request by nunegabriel**
In this project we did the following
* created  a react app
* Used React components.
* Used React props.
* Used React Router.
* Connect React and Redux.
* used the spaceX api
* Handled events in a React app.
* Wrote unit tests with React Testing Library.
* Use styles in a React app.
* Use React life cycle methods.
* Apply React best practices and language style guides in code.
* Use store, actions and reducers in React.

### CODE REVIEW

File: src/redux/rockets/rocketSlice.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+      .addCase(fetchRockets.rejected, (state) => {
+        const IsRejected = state;
+        IsRejected.status = 'rejected';
+      });
+  },
+});
+
+export const { bookRockets } = slice.actions;
+export default slice.reducer;
```
Comment (muneebulrehman) [inline src/redux/rockets/rocketSlice.js:64]: When I load the application, two rockets are already reserved. Kindly see the [attached video](https://www.loom.com/share/59d90e2ec731435ca7fd8ea8fd545e49) where I tried to describe the problem in detail.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,221 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```
Comment (muneebulrehman) [inline README.md:2]: Kindly make sure you have added the section `Future Features`, including 1-3 potential features to be added. All the main headers from this [README template](https://github.com/microverseinc/readme-template) should be present in your repo. The only optional section is FAQ.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Good design.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @nunegabriel & @Nejjari001 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**